freqtrade/freqai/prediction_models/CatboostClassifier.py

```diff
@@ -32,8 +32,9 @@ def fit(self, data_dictionary: Dict, dk: FreqaiDataKitchen, **kwargs) -> Any:
         )
 
         cbr = CatBoostClassifier(
-            allow_writing_files=False,
+            allow_writing_files=True,
```

Comment:
@robcaulk didn't we intentionally disable this setting as catboost then started to write random stuff in the current directory?

We can surely enable this if necessary - but we need to make sure it'll only write to the proper directory (the model directory) - not in random directories.

Reply:
Yes, but since here we set `train_dir`,  it directs the output to that directory (the model directory).